data/settings.py

```diff
@@ -10,7 +10,7 @@
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.getenv("DEBUG", False))
+DEBUG = True if (os.getenv('DEBUG') == 'true') else False
```

Comment:
It looks a bit flaky to me, what if I set `DEBUG=1`?
I prefer using `django-environ` [bool type](https://django-environ.readthedocs.io/en/latest/#supported-types).